src/mono/mono/metadata/custom-attrs.c

```diff
@@ -709,6 +709,10 @@ mono_custom_attrs_from_builders_handle (MonoImage *alloc_img, MonoImage *image,
 		MONO_HANDLE_ARRAY_GETREF (cattr, cattrs, i);
 		if (!custom_attr_visible (image, cattr, ctor_handle, &ctor_method))
 			continue;
+
+		if (image_is_dynamic(image))
```

Comment:
```suggestion
		if (image_is_dynamic (image))
```
nit: space after paren